test/form.test.ts

```diff
@@ -246,6 +246,26 @@ describe("Form: handleSubmit", () => {
   });
 });
 
+it("Return true when the inicial value is equal that value", () => {
+  const initialValues = { name: "Lorem", last: "Ipsum" };
+    const form = newForm<typeof initialValues>({
+      initialValues,
+      onSubmit: vi.fn(),
+    });
+
+    form.handleChange({
+      target: {
+        name: "name",
+        value: "Testing!",
+        type: "text",
+      },
+    } as unknown as Event);
+    const values = get(form.values);
+
+    expect(form.isDirty).toStrictEqual(true);
```

Comment:
Keep in mind `form.isDirty` is a Readable Store. That means you need to subscribe or
use a helper function to retrieve its value.

I recommend using: `get` instead, similar to `values` on L263.

```suggestion
    expect(get(form.isDirty)).toStrictEqual(true);
```